trackintel/io/from_geopandas.py

```diff
@@ -0,0 +1,203 @@
+    mapper : dict, further columns that should be renamed. 
+
+    Returns
+    -------
+    gdf : a validated triplegs GeoDataFrame
+
+    """
+  
+    
+    columns = {started_at:'started_at',
+               finished_at:'finished_at',
+               user_id:'user_id',
+               geom:'geom'}
+    columns.update(mapper)
+    
+    tpl = gdf.rename(columns=columns)
+    tpl = tpl.set_geometry('geom')
+    
+    assert tpl.as_triplegs
+    
+
+    return tpl
+
+def trips_from_gpd(gdf, started_at='started_at', finished_at='finished_at', user_id='user_id', origin_staypoint_id='origin_staypoint_id', destination_staypoint_id='destination_staypoint_id', mapper={}):
+    """
+    warps the pd.rename function to simplify the import of GeoDataFrames
+
+    Parameters
+    ----------
+    gdf : GeoDataFrame with valid geometry, containing the trips to import
+    started_at : str, name of the column storing the starttime of the staypoints. The default is 'started_at'.
+    finished_at : str, name of the column storing the endtime of the staypoints. The default is 'finished_at'.
+    user_id : str, name of the column storing the user_id. The default is 'user_id'.
+    origin_staypoint_id : str, name of the column storing the staypoint_id of the start of the tripleg
+    destination_staypoint_id : str, name of the column storing the staypoint_id of the end of the tripleg
+    mapper : dict, further columns that should be renamed. 
+
+    Returns
+    -------
+    gdf : a validated trips GeoDataFrame
+
+    """
+  
+    
+    columns = {started_at:'started_at',
+               finished_at:'finished_at',
+               user_id:'user_id',
+               origin_staypoint_id:'origin_staypoint_id',
+               destination_staypoint_id:'destination_staypoint_id'
+               }
+    columns.update(mapper)
+    
+    tps = gdf.rename(columns=columns)
+    
+    assert tps.as_trips
+    
+
+    return tps
+
+def locations_from_gpd(gdf, user_id='user_id', center='center', mapper={}):
+    """
+    warps the pd.rename function to simplify the import of GeoDataFrames
+
+    Parameters
+    ----------
+    gdf : GeoDataFrame with valid point geometry, containing the locations to import
+    user_id : str, name of the column storing the user_id. The default is 'user_id'.
+    center : str, name of the column storing the geometry (Center of the location). The default is 'center'.
+    mapper : dict, further columns that should be renamed. 
+
+    Returns
+    -------
+    gdf : a validated locations GeoDataFrame
+
+    """
+  
+    
+    columns = {user_id:'user_id',
+               center:'center'}
+    columns.update(mapper)
+    
+    lcs = gdf.rename(columns=columns)
+    lcs = lcs.set_geometry('center')
+    
+    assert lcs.as_locations
+    
+
+    return lcs
+
+def tours_from_gpd(gdf, user_id='user_id',started_at='started_at', finished_at='finished_at', origin_destination_location_id='origin_destination_location_id', journey='journey', mapper={}):
```

Comment:
for this function, there are no tests available? Maybe add an issue saying that test for tours are missing in both read_csv and from_geopandas